config/meetings.json

```diff
@@ -173,5 +173,10 @@
     "url": "https://github.com/asyncapi/community/issues/1177",
     "banner": "",
     "date": "2024-04-23T18:00:00.000Z"
+  },
+  {
+    "title": "Maintainers Growth Working Group Meeting",
+    "calLink": "https://www.google.com/calendar/event?eid=NGZ0NWZiNWw0MzJsbGR0M2Ewdm5kcGNkYzEgY19xOXRzZWlnbG9tZHNqNm5qdWh2YnB0czExY0Bn",
+    "date": "2024-04-22T14:00:00.000Z"
```

Comment:
@akshatnema There's no `banner` and `url` key in this unlike other objects. Can this be the reason for the lighthouse SEO check failure?